File: Hacking with macOS/Bindings-objc/Bindings-objc/ViewController.m
//
//  ViewController.m
//  Bindings-objc
//
//  Created by Jakub Charvat on 17/06/2020.
//  Copyright © 2020 jakcharvat. All rights reserved.
//

#import "ViewController.h"

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    NSArray *numbers = @[ @1, @2, @3, @4, @9 ];
    NSLog(@"%@", [numbers valueForKey:@"@count"]);
}


- (void)setRepresentedObject:(id)representedObject {
    [super setRepresentedObject:representedObject];

    // Update the view, if already loaded.
}


@end

File: Hacking with macOS/Bookworm/Bookworm/ViewController.swift
//
//  ViewController.swift
//  Bookworm
//
//  Created by Jakub Charvat on 17/06/2020.
//  Copyright © 2020 jakcharvat. All rights reserved.
//

import Cocoa

class ViewController: NSViewController {
    
    @objc dynamic var reviews = [Review]()

    override func viewDidLoad() {
        super.viewDidLoad()
    }
    
    override func viewDidAppear() {
        super.viewDidAppear()
        view.window?.isMovableByWindowBackground = true
    }

    override var representedObject: Any? {
        didSet {
        // Update the view, if already loaded.
        }
    }


}

